packages/toolkit/core-kit/src/regex.ts

```diff
@@ -7,3 +7,5 @@ export const mobileUriSchemeProtocolRegEx = /^[a-z][\d+_a-z-]*(\.[\d+_a-z-]+)+:$
 export const hexColorRegEx = /^#[\da-f]{3}([\da-f]{3})?$/i;
 export const dateRegex = /^\d{4}(-\d{2}){2}/;
 export const noSpaceRegEx = /^\S+$/;
+export const subdomainRegEx =
```

Comment:
There is a similar one in "shared"

Reply:
Let me check..

Reply:
This input field requires full domain URL that has at least 3 parts, e.g. foo.bar.com. But I think I need to change the name of the regex from `subdomainRegEx` to `customDomainRegEx` or `domainRegEx`